cli/assets/model/job-model-as-input.yml

```diff
@@ -0,0 +1,17 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandJob.schema.json
+
+# Possible Paths for models:
+# AzureML Datastore: azureml://datastores/<datastore-name>/paths/<path_on_datastore>
+# MLflow run: runs:/<run-id>/<path-to-model-relative-to-the-root-of-the-artifact-location>
+# Job: azureml://jobs/<job-name>/outputs/<output-name>/paths/<path-to-model-relative-to-the-named-output-location>
+# Model Asset: azureml:<my_model>:<version>
+
+command: |
+  ls ${{inputs.my_model}}
+code: load_score.py 
+inputs:
+  my_model:
+    type: mlflow_model # mlflow_model,custom_model, triton_model
```

Comment:
nit: leaving these examples here may give impression these are all the enum values, and if we add types in future this example would need to be updated. We should just point them to model schema?